src/common/window-utils.ts

```diff
@@ -22,6 +22,10 @@ export class WindowUtils {
         return window.setInterval(handler, timeout);
     }
 
+    public createObjectURL(ob: Blob | File | MediaSource): string {
```

Comment:
I feel we can rename `ob` to something that conveys more meaning. `sourceObject` maybe? thoughts?